<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>166</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="serverAddressLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>96</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Server Address: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="serverAddress">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>10</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="portBox">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>38</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="minimum">
    <number>1024</number>
   </property>
   <property name="maximum">
    <number>65525</number>
   </property>
   <property name="value">
    <number>6000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>29</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>User Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userNameEdit">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>67</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
